notebooks/notebooks/iclr2022/win-rate-vs-training-steps.ipynb

```diff
@@ -0,0 +1,366 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #54.        df19 = df.loc[df.board_size == 19]
</pre><p>Nit but inclined to make the board size a parameter, that defaults to 19, as we might in fact want to look at win rates of different board sizes in future. Also always nice to get rid of magic numbers in code.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/HumanCompatibleAI/KataGoVisualizer/pull/21/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>